app/src/main/java/org/mozilla/focus/utils/Features.kt

```diff
@@ -9,20 +9,4 @@ package org.mozilla.focus.utils
  */
 object Features {
     const val SEARCH_TERMS_OR_URL: Boolean = true
-    val IS_TRACKING_PROTECTION_CFR_ENABLED = AppConstants.isDevOrNightlyBuild
-    val IS_TOOLTIP_FOR_PRIVACY_SECURITY_SETTINGS_SCREEN_ENABLED = AppConstants.isDevOrNightlyBuild
-    const val ARE_HOME_PAGE_PRO_TIPS_ENABLED = true
-
-    /**
-     * Erase tabs CFR
-     */
-    var IS_ERASE_CFR_ENABLED = AppConstants.isDevOrNightlyBuild
-
-    /**
-     * Replace the current [FirstrunFragment.kt] which has multiple cards with
-     * [OnboardingFragment.kt] which has a new design
-     * When this flag it will be removed all classes, methods, tests, etc. which
-     * has "firstrun" in their name should be renamed to use "onboarding"
-     */
-    var ONBOARDING = AppConstants.isDevOrNightlyBuild
```

Comment:
Do all these flags need to go in to the `nimbus.fml.yaml` file?

Reply:
All of them can be covered by `onboarding` feature from` nimbus.fml.yaml` file